cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -82,3 +86,49 @@ def is_ansible_pod_stopped(enabled_embedded_appliance):
         assert wait_for(is_nginx_stopped, func_args=[enabled_embedded_appliance], num_sec=30)
     else:
         assert wait_for(is_ansible_pod_stopped, func_args=[enabled_embedded_appliance], num_sec=180)
+
+
+@pytest.mark.tier(1)
+def test_embed_tower_event_catcher_process(appliance, enabled_embedded_appliance):
+    """
+    EventCatcher process is started after Ansible role is enabled (rails
+    evm:status)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/4h
+        tags: ansible_embed
+    """
+    result = appliance.ssh_client.run_command(
+        "cd /var/www/miq/vmdb; rake evm:status | grep 'EmbeddedAnsible'"
+    ).output
+
+    checks = [
+        "EmbeddedAnsible",
+        "EmbeddedAnsible::Automation::EventCatcher",
+        "EmbeddedAnsible::Automation::Refresh",
+    ]
+
+    for data in result.splitlines():
+        for i in checks:
```

Comment:
I don't think after grepping you need checks for `EmbeddedAnsible`? is there any evm process for which you don't want to check status?

Simply you can check like
```
for row in result.splitlines(): 
    assert "started" in row 
```